packages/react/src/themes/teams/components/Dialog/dialogVariables.ts

```diff
@@ -26,4 +28,6 @@ export default (siteVariables): DialogVariables => ({
 
   overlayBackground: 'rgba(37, 36, 36, .75)', // todo: update to a palette value when daisy has mapped one
   overlayZIndex: 1000,
+
+  headerActionMargin: `${pxToRem(-3)} ${pxToRem(-8)} 0 0`,
```

Comment:
I understand why this is done and I had the same issue a few times. `<Button iconOnly text />` has some default padding that breaks most of the layouts where it's used.

**Should we add a `fitted` prop on the `Button` to remove that?**

On the other side, because of the negative margin, it looks a bit strange when we have the focus border around it:

![Screenshot 2019-07-12 at 12 07 52](https://user-images.githubusercontent.com/5442794/61121030-897d7780-a49e-11e9-81c1-6e933f0de4e4.png)

So my question is, what is expected here?
- align the icon within the button with the rest of the layout in regular mode? (what @mnajdova did here)
- align the button itself so that it looks good in focus mode?

fyi @codepretty

Reply:
@Bugaa92 I am not sure whether we should change it, because this is exactly how the focus styles should look by design:

![image](https://user-images.githubusercontent.com/4512430/61121329-3f48c600-a49f-11e9-9eb8-69440e5a1672.png)

I agree that it looks strange, but would rather not make any changes at this moment. If this would have been unexpected behavior, then yes, I agree that we should have reconsidered this, but at this moment it is consistent with the designs...

Reply:
got it, thanks for the explanation @mnajdova makes sense